packages/tools/env/rubygems/start.coffee

```diff
@@ -67,5 +67,13 @@ nikita
       cmd: """
       yum install -y gcc ruby ruby-devel
       """
+    @lxd.exec
+      header: 'Install Atom'
+      container: options.container
+      cmd: """
+      rpm --import https://packagecloud.io/AtomEditor/atom/gpgkey
+      sh -c 'echo -e "[Atom]\nname=Atom Editor\nbaseurl=https://packagecloud.io/AtomEditor/atom/el/7/\$basearch\nenabled=1\ngpgcheck=0\nrepo_gpgcheck=1\ngpgkey=https://packagecloud.io/AtomEditor/atom/gpgkey" > /etc/yum.repos.d/atom.repo'
+      yum install atom-0:1.46.0-0.1.x86_64
+      """
```

Comment:
npm executes tests in the tools-rubygems container, so I installed apm in the same container to execute apm tests there too. My reasoning was that it means just a few lines of extra code rather than new files with a lot of duplicated code. It also is analagous to what npm does.

Reply:
where did you saw this, I don't see any occurrence of npm inside ruby and if there was it would have been a mistake. npm executes its tests in the tools/env/npm container, it has nothing to do with rubygems